apsuite/commisslib/measure_tbt_data.py

```diff
@@ -0,0 +1,217 @@
+"""."""
+import numpy as _np
+from scipy.optimize import curve_fit as _curve_fit
+import matplotlib.pyplot as _mplt
+import datetime as _datetime
+from siriuspy.devices import PowerSupplyPU
+
+from .meas_bpms_signals import AcqBPMsSignals as _AcqBPMsSignals, \
+    AcqBPMsSignalsParams as _AcqBPMsSignalsParams
+
+class TbTDataAnalysis(_AcqBPMsSignals):
```

Comment:
We typically write the classes following the workflow: Params -> Measurement -> Analysis. It is just a matter of organization, it doesn't affect how the classes work.

Reply:
Thanks for the feedback, Murilo. I hadn't yet checked out the acquisition script you mentioned friday. I did it this morning and brought some patterns from there, as well as working on you other suggestions. Thanks!